VirtoCommerce.DynamicExpressionsModule.Data/Promotion/Conditions/CatalogConditions/ConditionEntryIs.cs

```diff
@@ -20,9 +23,18 @@ public class ConditionEntryIs : DynamicExpression, IConditionExpression
         {
             var paramX = linq.Expression.Parameter(typeof(IEvaluationContext), "x");
             var castOp = linq.Expression.MakeUnary(linq.ExpressionType.Convert, paramX, typeof(PromotionEvaluationContext));
-            var methodInfo = typeof(PromotionEvaluationContextExtension).GetMethod("IsItemInProduct");
+            linq.MethodCallExpression methodCall = null;
+            if (ProductIds != null)
+            {
+                var methodInfo = typeof(PromotionEvaluationContextExtension).GetMethod("IsItemInProducts");
```

Comment:
We could use [existing extension method](https://github.com/VirtoCommerce/vc-module-dynamic-expressions/blob/916856ee3b2431ab3f9f363b6b702654ac1a4886/VirtoCommerce.DynamicExpressionsModule.Data/Common/Extensions/ExpressionExtensions.cs#L9) with the same functionality here.